test/getCommands.spec.js

```diff
@@ -7,13 +7,15 @@ const mockFs = require('mock-fs');
 const sinon = require('sinon');
 
 const commands = require('./fixtures/commands');
-const testPluginPath = path.join(process.cwd(), 'node_modules', 'rnpm-plugin-test');
-const testPluginPath2 = path.join(process.cwd(), 'node_modules', 'rnpm-plugin-test-2');
+const nestedPluginPath = path.join(process.cwd(), 'node_modules', 'rnpm-plugin-test');
```

Comment:
Can we change all the paths to be `process.cwd(), 'node_modules', 'rnpm', 'node_modules', 'rnpm-plugin-test'` and `process.cwd(), 'node_modules', 'rnpm-plugin-test'` to avoid confusion? Currently it's quite tricky (my fault).